apps/trading/components/constants.ts

```diff
@@ -1,3 +1,3 @@
 export const DEBOUNCE_UPDATE_TIME = 500;
 export const TOKEN_DEFAULT_DOMAIN = 'https://token.fairground.wtf';
-export const NEW_PROPOSAL_LINK = '/governance/propose';
+export const NEW_PROPOSAL_LINK = '/governance/propose/new-market';
```

Comment:
chore (non-blocking): badly named, NEW_MARKET_PROPOSAL_LINK IMO would be better, know it's not from your PR but new proposal is more general IMO 👍

Reply:
it's already merged, but I will remember it to fix in the future